public/js/components/stories/Frames.js

```diff
@@ -1,46 +1,26 @@
 "use strict";
 
 const React = require("react");
-const { DOM: dom, createElement } = React;
-const { Provider } = require("react-redux");
-const { Map, fromJS } = require("immutable");
+const { DOM: dom, createElement, createFactory } = React;
+const { renderComponent, storiesOf } = require("./utils");
 
-const { createStore } = require("./utils");
+const Frames = require("../Frames");
 
-const { storiesOf } = require("@kadira/storybook");
-const Frames = React.createFactory(require("../Frames"));
-const fixtures = require("../../test/fixtures");
+const style = {
+  width: "300px",
+  margin: "auto",
+  paddingTop: "100px" };
 
-function getData(fixtureName) {
-  const fixture = fixtures[fixtureName];
-  const frames = fixture.pause.frames;
-  return {
-    pause: Map({ frames }),
-    sources: fromJS({ sources: fixture.sources.sources })
-  };
+const component = dom.div(
+  { className: "accordion" },
+  dom.div({ className: "_content" }, createElement(createFactory(Frames)))
+);
+
+function renderFrames(fixtureName) {
+  return renderComponent(component, fixtureName, { style })
 }
 
 storiesOf("Frames", module)
-  .add("Blank", () => {
-    return renderContainer(Frames);
-  })
-  .add("TodoMVC Todo Toggle", () => {
-    return renderContainer(Frames, getData("todomvcToggle"));
-  })
-  .add("Nested Closures", () => {
-    return renderContainer(Frames, getData("pythagorean"));
-  });
-
-function renderContainer(Component, data) {
-  const store = createStore(data);
-  return dom.div(
-    { style: {
-      width: "400px",
-      margin: "auto",
-      paddingTop: "100px"
-    }},
-    createElement(Provider, { store },
-      dom.div({ className: "accordion" },
-        dom.div({ className: "_content" }, createElement(Component))))
-  );
-}
+  .add("Not Paused", () => renderFrames("todomvc"))
+  .add("Event Handler", () => renderFrames("todomvcUpdateOnEnter"))
+  .add("Nested Closures", () => renderFrames("pythagorean"));
```

Comment:
This feels like cheating :) I was thinking that stories use the raw "dumb" component and pass things in as documentation for what props the component uses. It would also allow us to develop the components in isolation.

However, I get that many of our debugger-specific components heavily rely on the pause data specifically so it's nice to just load that in. I'll try to use storybook more, as I don't have a strong opinion about this now.